apps/src/templates/rubrics/LearningGoals.jsx

```diff
@@ -0,0 +1,392 @@
+              {UNDERSTANDING_LEVEL_STRINGS[displayUnderstanding]}
+            </BodyThreeText>
+          )}
+          {submittedEvaluation && (
+            <div className={style.submittedEvaluation}>
+              {submittedEvaluation.understanding !== null && (
+                <BodyThreeText>
+                  {
+                    UNDERSTANDING_LEVEL_STRINGS[
+                      submittedEvaluation.understanding
+                    ]
+                  }
+                </BodyThreeText>
+              )}
+              {submittedEvaluation.feedback && (
+                <FontAwesome
+                  icon="message"
+                  className="fa-regular"
+                  title={i18n.feedback()}
+                />
+              )}
+            </div>
+          )}
+        </div>
+        <button
+          type="button"
+          className={style.learningGoalButton}
+          onClick={() => onCarouselPress(1)}
+        >
+          <FontAwesome icon="angle-right" />
+        </button>
+      </div>
+
+      {/*TODO: Pass through data to child component*/}
+      <div>
+        {teacherHasEnabledAi &&
+          !!studentLevelInfo &&
+          !!aiEvalInfo &&
+          aiUnderstanding !== undefined && (
+            <div className={style.openedAiAssessment}>
+              <AiAssessment
+                isAiAssessed={learningGoals[currentLearningGoal].aiEnabled}
+                studentName={studentLevelInfo.name}
+                aiConfidence={aiConfidence}
+                aiUnderstandingLevel={aiUnderstanding}
+                aiEvalInfo={aiEvalInfo}
+              />
+            </div>
+          )}
+        <div className={style.learningGoalExpanded}>
+          {!!submittedEvaluation && renderSubmittedFeedbackTextbox()}
+          <EvidenceLevels
+            learningGoalKey={learningGoals[currentLearningGoal].key}
+            evidenceLevels={learningGoals[currentLearningGoal].evidenceLevels}
+            canProvideFeedback={canProvideFeedback}
+            understanding={displayUnderstanding}
+            radioButtonCallback={radioButtonCallback}
+            submittedEvaluation={submittedEvaluation}
+            isStudent={isStudent}
+          />
+          {learningGoals[currentLearningGoal].tips && !isStudent && (
+            <div>
+              <Heading6>{i18n.tipsForEvaluation()}</Heading6>
+              <div className={style.learningGoalTips}>
+                <SafeMarkdown
+                  markdown={learningGoals[currentLearningGoal].tips}
+                />
+              </div>
+            </div>
+          )}
+          {!!studentLevelInfo && renderAutoSaveTextbox()}
+        </div>
+      </div>
+    </div>
+  );
+}
+
+LearningGoals.propTypes = {
+  teacherHasEnabledAi: PropTypes.bool,
+  canProvideFeedback: PropTypes.bool,
+  reportingData: reportingDataShape,
+  studentLevelInfo: studentLevelInfoShape,
+  submittedEvaluation: submittedEvaluationShape,
+  isStudent: PropTypes.bool,
```

Comment:
This only seems to be called with `false` at the moment. I assume there's plans to show this to students eventually? Could simplify your logic a bit if not.

Reply:
That needs to stay. We have separate components for students and teachers further down the hierarchy and this determines which are displayed.